messaging/websockets-client/pom.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>io.quarkus.ts.qe</groupId>
+        <artifactId>parent</artifactId>
+        <version>1.0.0-SNAPSHOT</version>
+        <relativePath>../..</relativePath>
+    </parent>
+    <artifactId>websockets-client</artifactId>
+    <name>Quarkus QE TS: Messaging: Websockets client</name>
+    <dependencies>
+        <dependency>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-websockets</artifactId>
```

Comment:
This is not correct as this dependency brings both server and client. And according this test: https://github.com/quarkus-qe/quarkus-test-suite/pull/273/files#diff-ea69dcb1d9ecd506ef36c6ffd128ff24529c5c1aa43b2c0bd820417d54d51843R13, you want to verify the client isolately.

Reply:
Added "provided" scope